action.yml

```diff
@@ -1,18 +1,24 @@
-name: 'Code Dx Analysis'
-description: 'Analyze your source code and binaries with Code Dx'
+name: 'SRM Analysis'
+description: 'Analyze your source code and binaries with SRM'
 inputs:
   # main config options
```

Comment:
This should be updated with any new parameters

Reply:
I think this file was already updated with the new parameters in the previous commit.